src/python/pants/task/testrunner_task_mixin.py

```diff
@@ -450,6 +451,31 @@ def chroot(self, targets, workdir):
         yield chroot
 
   def _execute(self, all_targets):
+
+    deprecated_conditional(
+      lambda: self.get_options().is_default("fast"),
+      removal_version="1.25.0.dev1",
+      entity_description="using the default value for --no-fast/--fast",
+      hint_message="Please explicitly set the flag --fast. The default will change from --fast to"
+                   "--no-fast in 1.25.0.dev1, and the option will be removed in 1.27.0.dev1. We "
+                   "recommend setting the option to --no-fast for better isolation of tests and to"
+                   "prepare for Pants eventually dropping the option."
+    )
+    deprecated_conditional(
+      lambda: self.get_options().is_default("chroot"),
+      removal_version="1.25.0.dev1",
+      entity_description="using the default value for --no-chroot/--chroot",
+      hint_message="Please explicitly set the flag --chroot. The default will change from "
+                   "--no-chroot to --chroot in 1.25.0.dev1, and the option will likely be removed in"
```

Comment:
This might be worth a docsite entry instead, that the option can link to?

Reply:
I don't know where that would go. Like this? https://www.pantsbuild.org/options_reference.html#test

Reply:
Going to merge but happy to add docs in a followup!